lib/memsize_helpers.rb

```diff
@@ -1,6 +1,40 @@
+require "objspace"
+
 module MemsizeHelpers
   def self.memsize_rss_in_kb
-    _, size = %x(ps -p "#{$$}" -o pid=,rss=).strip.split.map(&:to_i)
-    size
+    %x(ps -p "#{$$}" -o rss=).strip.to_i
+  end
+
+  def self.reset!
+    @memsize_rss_start = memsize_rss_in_kb
+  end
+  reset!
+
+  def self.check_usage
+    GC.start
```

Comment:
I'm not sure if we really want to do this in a benchmark. On the one hand, it makes the results more deterministic, but on the other hand, I'm quite interested in how these changes impact things in practice. The `GC.start` call only existed in the various "leak*.rb" scripts so we could more closely see what the current Ruby heap size is.

Reply:
Ok, I wasn't sure if including this made it more or less representative of a real app.
I can gate this with the `verbose?` check.